first pass at ozunconf draft post

njtierney opened the pull request:
This is the first pass at the ozunconf blog post, some things I need to add:

- twitter image
- photos from the ozunconf

There are also a few grammar / style things that I would like to improve upon - i'm not sure if there are other ideas that I can expand upon in this blog post, or if the current summary is OK.

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+    url: https://www.njtierney.com
+    twitter: nj_tierney
+# twitterImg: img/blog-images/2019-01-03-ozunconf18/fun-photo.jpg
+date: 2019-01-03
```
date: 2019-01-08
in YAML, path to photos, filename etc pls

njtierney replied:
OK no worries - do I need to change the filename to be "2019-01-08..." intead of "2019-01-03..." ?

stefaniebutland replied:
yes please

njtierney replied:
OK done!

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+categories: blog
+topicid: 925
+tags:
+- r
```
for tags add
- unconf
- ozunconf
- ozunconf18

delete "925" from topicid and I will add topicid at publication time

njtierney replied:
Done!

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+
+[summary photo]
+
+In late November 2018, we ran the [third annual rOpenSci ozunconf](http://ozunconf18.ropensci.org). This is the sibling rOpenSci unconference, held in Australia. We ran the first ozunconf in 2016, [in Brisbane](http://auunconf.ropensci.org), the second in 2017 [we ran it in Melbourne](http://ozunconf17.ropensci.org).
```
You could link to your summary posts for richer info, rather than the websites
https://ropensci.org/blog/2017/10/31/ozunconf2017/
https://ropensci.org/blog/2016/06/16/au-unconf/

Also fyi, you can get all posts from all ozunconfs with https://ropensci.org/tags/ozunconf/, all ozunconf17 posts with https://ropensci.org/tags/ozunconf17/

njtierney replied:
Awesome, thanks! I didn't know about the tags like that, that's really neat!

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+As usual, before the unconf, we started discussion on [github issue threads](https://github.com/ropensci/ozunconf18/issues), 
+and the excitement was building with the number of issues.
+
+The day before the unconf we ran "Day 0 training" - an afternoon explaining R packages and github. This training is run a little different to a typical course on R packages and github, in that the aim is to get people acquainted with what R packages and GitHub are, that they can learn the material, and that it is worthwhile to learn. We showed people how to create an R pacakge, set it up with ver ...
```
github -> GitHub throughout

njtierney replied:
Done

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+As usual, before the unconf, we started discussion on [github issue threads](https://github.com/ropensci/ozunconf18/issues), 
+and the excitement was building with the number of issues.
+
+The day before the unconf we ran "Day 0 training" - an afternoon explaining R packages and github. This training is run a little different to a typical course on R packages and github, in that the aim is to get people acquainted with what R packages and GitHub are, that they can learn the material, and that it is worthwhile to learn. We showed people how to create an R pacakge, set it up with ver ...
```
can you clarify this line "a typical course..." since you lay out your unique content but don't point out what the typical content would be

(I get what you're trying to convey - just needs revision)

njtierney replied:
I've tried to change this now with a slightly differnet structure -  let me know what you think?

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+
+[photo of day 0 training]
+
+Day 1 started with a bang, in the beautiful north arts building at the university of Melbourne, and dusted off the tried and tested rOpenSci icebreaker. We then moved on to choose projects. Everyone moved around the room, exploring options, and we settled into groups.
```
Please link to post on ice breaker now that we've published it: `[rOpenSci icebreaker](https://ropensci.org/blog/2018/11/01/icebreaker/)`

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+
+[photo]
+
+On the second day, [Adam Gruer](https://twitter.com/AdamGruer) led a fireside chat on how to continue the experience of the rOpenSci ozunconf after we wrap up. Among the suggestions were starting or joining coding clubs at your university or place of work. Adam also made the point that a reason so many people enjoy the unconf is that freeing up your time and expectations has the surprising effect ...
```
revise "fireside chat on how to continue" to indicate it was not to prescribe a how to, but to share ideas e.g. "fireside chat on how we might continue..."

stefaniebutland replied:
link to Robbie Bonelli?

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+
+> We gain personal knowledge that we would never have gained working on our own, even if we never use or work on these things again. Make it about learning something new, this is how to break down the idea of having to get something out of this for your job.
+
+We closed up the conference with 3 minute presentations from everyone on what they got up to over the past two days. 
```
3-minute (add hyphen)

stefaniebutland commented on content/blog/2019-01-03-ozunconf18.md:
```diff
@@ -0,0 +1,54 @@
+
+[presentation photos]
+
+Looking back, it really is amazing how much can get done over a just a few days!  We couldn't have gotten it done without the help of our sponsors, Microsoft Azure, The R Consortium, RStudio, The University of Melbourne, and The Monash Business School.
```
link to sponsors (I know you prob plan to do that)

njtierney commented:
I've got some more photos that I can link to, I've sent you a DM with the folder, other than that, I've added a sentence or two on git commit messages that I remembered Adam mentioned at the unconf that I really thought was worthwhile sharing.

I've also added a link to the Credibly curious podcast episode - let me know if you want me to move this to the end or somewhere else :)

I think that's nearly it, let me know if there is anything you'd like me to add :)

stefaniebutland commented on content/blog/2019-01-08-ozunconf18.md:
```diff
@@ -0,0 +1,76 @@
+
+![](/img/blog-images/2019-01-08-ozunconf/saras-felix-miles.jpeg)
+
+Day 1 started with a bang, in the beautiful north arts building at the university of Melbourne, and dusted off the tried and tested [rOpenSci icebreaker](https://ropensci.org/blog/2018/11/01/icebreaker/). We then moved on to choose projects. Everyone moved around the room, exploring options, and we settled into groups.
```
```suggestion
Day 1 started with a bang, in the beautiful north arts building at the University of Melbourne, and dusted off the tried and tested [rOpenSci icebreaker](https://ropensci.org/blog/2018/11/01/icebreaker/). We then moved on to choose projects. Everyone moved around the room, exploring options, and we settled into groups.
```

stefaniebutland commented on content/blog/2019-01-08-ozunconf18.md:
```diff
@@ -0,0 +1,76 @@
+
+![](/img/blog-images/2019-01-08-ozunconf/mitch-rob-emi.jpeg)
+
+Throughout the conference, [Saskia Freytag](https://twitter.com/trashystats) recorded conversations to capture the event. These were edited into the podcast ["Credibly Curious"](https://soundcloud.com/crediblycurious), in episode 6, ["Ozunconf"](https://soundcloud.com/crediblycurious/episode-6-oz-unconf), run by Saskia and Myself.
```
```suggestion
Throughout the conference, [Saskia Freytag](https://twitter.com/trashystats) recorded conversations to capture the event. These were edited into the podcast ["Credibly Curious"](https://soundcloud.com/crediblycurious), in episode 6, ["Ozunconf"](https://soundcloud.com/crediblycurious/episode-6-oz-unconf), run by Saskia and me.
```

stefaniebutland commented on content/blog/2019-01-08-ozunconf18.md:
```diff
@@ -0,0 +1,76 @@
+
+![](/img/blog-images/2019-01-08-ozunconf/melb-logo.png)
+
+In late November 2018, we ran the [third annual rOpenSci ozunconf](http://ozunconf18.ropensci.org). This is the sibling rOpenSci unconference, held in Australia. We ran the first ozunconf in 2016, [in Brisbane](ropensci.org/blog/2016/06/16/au-unconf), the second in 2017 [we ran it in Melbourne](ropensci.org/blog/2017/10/31/ozunconf2017).
```
```suggestion
In late November 2018, we ran the [third annual rOpenSci ozunconf](http://ozunconf18.ropensci.org). This is the sibling rOpenSci unconference, held in Australia. We ran the first ozunconf [in Brisbane](https://ropensci.org/blog/2016/06/16/au-unconf) in 2016, and the second [in Melbourne](https://ropensci.org/blog/2017/10/31/ozunconf2017) in 2017.
```

njtierney commented:
Thanks for the feedback and commits, Stef! (That is so neat that you can add commits like that now!)

Some responses:

> "comment on using from Adam Gruer " - missing word? do you mean "using GitHub"??

**yes** - thanks for picking up on this, I added the word "git".

> "This afternoon course was developed last year by myself and Roger Peng. This year, Saras Windecker stepped up to the plate and did an incredible job teaching." Any chance you could re-order this to say Saras taught the thing you developed last year? e.g. "This year, Saras Windecker stepped up to the plate and did an incredible job teaching the afternoon course that Roger Peng and I developed last year"

**yes, thank you for this - fixed now**

> Unfortunately the pull quote "This commit will …" looks awkward, because my eyes are drawn to it before I even read the preceding sentence to understand it. It's especially distracting because I expect a pull quote to be about ozunconf itself :-(. Keep without pull quote?

**I've kept this in text but removed the pull quote - does that read OK?**

> please add pic/photo credits: if you took all the pics, could add one credit below the first pic

**I've added the following, is this OK?**

```
![](/img/blog-images/2019-01-08-ozunconf/adam-anna-sarah.jpeg)
_Photos taken by Ajay from [Fotoholics](https://www.fotoholics.org/)_
```

> for the Australia art images, I think you'll need to add credits below

** These photos were taken and modified from last years logo. They are a bit old now, I don't actually have the credits handy :/**
